src/nvme/fabrics.c

```diff
@@ -677,14 +705,14 @@ nvme_ctrl_t nvmf_connect_disc_entry(nvme_host_t h,
 	nvme_ctrl_t c;
 	int ret;
 
+	strchomp_discovery_log_entry(e);
```

Comment:
I think we can even drop this one here. The passed entry should already be sanitized.

Reply:
Drop this.